### DISCUSSION TITLE
In Docker Compose, allow for definition of backup volume, folder, and environment variable

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2191] [Ideas] [answered]
nextcloud_aio_backupdir:
    name: nextcloud_aio_backupdir
    driver: local
    driver_opts:
      o: bind
      type: none
      device: /dockerbackup

    environment:
      - NEXTCLOUD_BACKUPDIR=nextcloud_aio_backupdir

**Comment by szaimen** [comment, marked as answer]
Hi, we have no intention to make this configurable via ENV since there already is a GUI option.

**Reply by FlattusBlastus** [reply]
but...docker compose......
